.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "OpenTimelineIO"]
 	path = OpenTimelineIO
 	url = https://github.com/PixarAnimationStudios/OpenTimelineIO
+[submodule "cmocka"]
```

Comment:
cmocka is new to me ~ could you add some notes, perhaps in the top level readme along the lines of "cmocka is a C mocking library, selected here to enable ... "